api/client/README.md

```diff
@@ -0,0 +1,227 @@
+### Introduction
```

Comment:
Thank you for these resources, this should help organize it better. I do want to keep something in the README because it can add some nice flavor to the top of the `pkg.go.dev`. See [aws-sdk](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2) for example.

How about I move the bulk of the docs to `goteleport.com/docs` as suggested, and then add abbreviated sections with links to the docs in the README. That way whether a user starts from `pkg.go.dev` or the main docs, they'll find all the documentation with ease.

Reply:
yes that works